docs/docsite/rst/community/collection_development_process.rst

```diff
@@ -153,7 +153,7 @@ A single changelog fragment may contain multiple sections but most will only con
 
 Each changelog entry must contain a link to its issue between parentheses at the end. If there is no corresponding issue, the entry must contain a link to the PR itself.
 
-Most changelog entries are ``bugfixes`` or ``minor_changes``.
+Most changelog entries are ``bugfixes`` or ``minor_changes``. You can use ``trivial`` as well for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.
```

Comment:
Most changelog entries are ``bugfixes`` or ``minor_changes``. You can also use ``trivial`` for any collection that requires a changelog fragment for each pull request. The ``trivial`` changelog fragments are excluded from the changelog output.
```suggestion
Most changelog entries are ``bugfixes`` or ``minor_changes``. You can use ``trivial`` as well for any collection that requires a changelog fragment for each pull request. ``trivial`` changelog fragments are excluded from the changelog output.
```